src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -47,6 +47,21 @@ public MigrateRepoCommand(OctoLogger log, ITargetGithubApiFactory targetGithubAp
                 IsRequired = false,
                 Description = "Defaults to the name of source-repo"
             };
+            var targetApiUrl = new Option<string>("--target-api-url")
```

Comment:
what scenario would this be used? migrating TO GHES? If so I think we need to call it out in the arg description.

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/6b41a079ac3c8f3ad659c9a3b7d41173252b9d94